### ISSUE TITLE
Custom validation set

### ISSUE THREAD

**Issue by meriliis** [interpretml/interpret#31]
Hi,
Would it be possible to add an option to specify the validation data in EBM fit() function instead of letting sklearn calculate it?

**Comment by interpret-ml** [comment]
Hi @meriliis,

Thanks for suggesting this. We agree that this definitely makes sense in many domains (ex: when train/validation/test are time-dependent). We don't have an immediate timeline on implementing this, but just wanted to let you know that it's on our backlog and something that we are interesting in adding in the future. 

Thanks!
-InterpretML Team

**Comment by paulbkoch** [comment]
Issue https://github.com/interpretml/interpret/issues/268 is a duplicate, but that issue has more discussion on the API implementation, so closing this issue in favor of tracking custom validation sets there.
